apps/voting/app/src/app-state-reducer.js

```diff
@@ -1,5 +1,6 @@
 import BN from 'bn.js'
 import { hasLoadedVoteSettings } from './vote-settings'
+import { Percentage, TokenAmount } from './math-utils'
```

Comment:
Hmm, we don't have to do it _now_, but I think we could use our newly released [TokenAmount](https://github.com/aragon/token-amount) lib!